src/v2/view-builder/utils/AuthenticatorUtil.js

```diff
@@ -130,6 +130,17 @@ const getAuthenticatorData = function (authenticator, isVerifyAuthenticator) {
       buttonDataSeAttr: getButtonDataSeAttr(authenticator),
     });
     break;
+
+  case AUTHENTICATOR_KEY.CUSTOM_OTP: {
+    Object.assign(authenticatorData, {
+      description: isVerifyAuthenticator
+        ? ''
+        : loc('oie.custom_otp.description', 'login'),
+      iconClassName: 'mfa-hotp',
```

Comment:
what `hotp` ?

Reply:
In V1 this was called `hotp`, which I think stands for hardware OTP. We are re-using the classname for the icon here.